cmd/kritis/admission/main.go

```diff
@@ -82,20 +95,28 @@ func NewServer(addr string) *http.Server {
 }
 
 func StartCronJob() error {
```

Comment:
exported function StartCronJob should have comment or be unexported

Reply:
done